spec/javascripts/bootstrap_sortable_rails_spec.js

```diff
@@ -1,17 +1,18 @@
 describe('gem bootstrap_sortable_rails', function() {
     var header;
     beforeEach(function() {
-        setFixtures('<table class="sortable"><thead><tr><th data-sortkey="0">Name</th></tr></thead>' +
-            '<tbody><tr><td data-value="Harrow Baptist Church"><a href="/organisations/1">Harrow Baptist Church</td></tr>' +
-            '<tr><td data-value="Human Touch Worldwide"><a href="/organisations/1">Human Touch Worldwide</td></tr></tbody></table>');
+        setFixtures('<table class="sortable"><thead><tr><th data-sortkey="0">Name</th></tr></thead><tbody>')
+        appendSetFixtures('<tr><td data-value="Harrow Baptist Church"><a href="/organisations/1">Harrow Baptist Church</td></tr>')
+        appendSetFixtures('<tr><td data-value="Human Touch Worldwide"><a href="/organisations/1">Human Touch Worldwide</td></tr>')
+        appendSetFixtures('</tbody></table>')
         header = $('th');
     });
     describe('when clicking on the tableheader', function() {
         it('sorts A-Z then Z-A', function() {
             header.click();
             expect($('td').first()).toHaveText('Harrow Baptist Church');
             header.click();
-            expect($('td').first()).toHaveText('Human Touch Worldwide')
+            expect($('td').last()).toHaveText('Human Touch Worldwide')
```

Comment:
@tansaku I did this change just to check that the problem was the sorting functionality, and yes, when we click the header the sorting isn't working :+1: .

Reply:
The functionality on the website is working, the one that is failing to mimic the behavior is the test itself.